test_crates/enum_struct_variant_field_missing/new/src/main.rs

```diff
@@ -0,0 +1,10 @@
+pub enum Enum {
+    FieldWillBeMissing {
+        foo: usize,
+    }
+}
+
+/// This struct variant should not be reported by the `enum_struct_variant_field_missing` rule:
+/// it will be removed altogether, so the correct rule to catch it is
+/// the `enum_variant_missing` rule and not the rule for missing fields.
```

Comment:
This comment is unclear in this incarnation, since it's referencing code that has already been deleted: there's no `this struct variant` here.

In the original context, it was fine because the struct variant was feature-flagged and present in the source file near this comment. Consider rewording it to be more appropriate to this context.